assets/js/components/GoogleChart.js

```diff
@@ -19,205 +19,186 @@
 /**
  * External dependencies
  */
-import debounce from 'lodash/debounce';
-import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Chart } from 'react-google-charts';
 
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { useEffect, useState, useRef } from '@wordpress/element';
+import { useEffect, useLayoutEffect, useRef } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import ProgressBar from './ProgressBar';
+import PreviewBlock from './PreviewBlock';
 
 export default function GoogleChart( props ) {
 	const {
-		chartID,
+		chartEvents,
```

Comment:
@tofumatt we also need to add prop types for this component as well.

Reply:
The `chartEvents` should be part of the propTypes from the existing GoogleChart that we pass through. If not I'll add them though 👍🏻

Reply:
@tofumatt the new GoogleChart component does not have propTypes at all. We need to add it.

Reply:
Oh no, I see. My editor had been doing weird removals of prop types sometimes and I don't know why. It seems they were removed here even though I thought I added them. I'll add them back! 😓

Reply:
@tofumatt I had a similar experience (on a PR that @eugene-manuilov reviewed and luckily he spotted the problem!)

I notice that our linting rules are very aggressive with removing unreachable code.  As opposed to just flagging it, it also autofixes it by deleting (which throws me off while I'm developing! :smiley: ).  I'm not sure if it's related to this :thinking:

Reply:
@danielgent Yup, I think that's it. I usually catch it, but it's easy to miss the proptypes being removed at the bottom of the page. 🤷🏻